database/seed.ts

```diff
@@ -0,0 +1,185 @@
+import {Client} from "pg"
+import {database} from "./index";
+import {company, condominium, employee, department, scheduling, user, userType} from "./schema"
+import {fakerPT_BR as faker} from "@faker-js/faker";
+
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./.env" });
+
+if (!("DATABASE_URL" in process.env))
+    throw new Error("DATABASE_URL not found in .env")
+
+function createData(){
+    const generateForEach: number = 5;
+
+    const employeeUserData: (typeof user.$inferInsert)[] = []
+    const condominiumUserData: (typeof user.$inferInsert)[] = []
+    const condominiumData: (typeof condominium.$inferInsert)[] = []
+    const companyData: (typeof company.$inferInsert)[] = []
+    const departmentData: (typeof department.$inferInsert)[] = []
+    const employeeData: (typeof employee.$inferInsert)[] = []
+    const schedulingData: (typeof scheduling.$inferInsert)[] = []
```

Comment:
As counterintuitive as it seems, `$inferSelect` is actually preferable for a better representation of the type annotations here. 

This is because, from TypeScript's perspective, if in Drizzle `schema.ts` we declare that those primary keys are `notNull()` but also `defaultRandom()`, the type inference understands that this value can be left as undefined, because it will be autogenerated by the database engine, which is technically a correct inference.

In the seeding script case however, we need to references to these ID's directly because we're sequentially creating relational data, so we need to reference created ID's in subsequent objects.

`$inferSelect` will always have `id: string` instead of `id?: string | undefined` because, from a selection perspective, an `id` will never be `undefined` (it was either defined at runtime, or by the database engine).

```suggestion
    const employeeUserData: (typeof user.$inferSelect)[] = []
    const condominiumUserData: (typeof user.$inferSelect)[] = []
    const condominiumData: (typeof condominium.$inferSelect)[] = []
    const companyData: (typeof company.$inferSelect)[] = []
    const departmentData: (typeof department.$inferSelect)[] = []
    const employeeData: (typeof employee.$inferSelect)[] = []
    const schedulingData: (typeof scheduling.$inferSelect)[] = []
```